notifier/notifier.go

```diff
@@ -231,6 +236,18 @@ func (n *Notifier) Send(alerts ...*model.Alert) {
 	n.setMore()
 }
 
+func (n *Notifier) relabelAlerts(alerts []*model.Alert) []*model.Alert {
+	var relabeledAlerts []*model.Alert
+	for _, alert := range alerts {
+		labels, _ := retrieval.Relabel(alert.Labels, n.opts.RelabelConfigs...)
```

Comment:
Yes you're right, I wanted to do that. Will move relabeling to it's own package, I think that makes sense as well.